common/power_bloc_identities/00_power_bloc_identities.txt

```diff
@@ -184,7 +185,7 @@ identity_sovereign_empire = {
 		power_bloc_invite_acceptance_major_power_add = -10
 		power_bloc_invite_acceptance_unrecognized_major_power_add = -10
 		power_bloc_invite_acceptance_add = -30
-		power_bloc_cohesion_per_member_add = -3
+		power_bloc_cohesion_per_member_add = -5
```

Comment:
Is this tested for Britain?

Reply:
The per-member modifier only affects non-subjects. Britain is entirely unaffected as a result. This is why it became necessary to deepen it in the first place, to encourage sovereign empires to do the subjugation they are supposed to.